rhscl-mysql-apb/Makefile

```diff
@@ -0,0 +1,89 @@
+CONTEXT = ansibleplaybookbundle
+VERSION = v0.1
+IMAGE_NAME = rhscl-mysql-apb
+TARGET = centos7
+REGISTRY = docker-registry.default.svc.cluster.local
+OC_USER = developer
+OC_PASS = developer
+APB_APP = mysql
+
+# Allow user to pass in OS build options
+ifeq ($(TARGET),rhel7)
+	DFILE := Dockerfile.${TARGET}
+else
+	DFILE := Dockerfile
+endif
+
+all: build
+build:
+	docker build -t ${CONTEXT}/${IMAGE_NAME}:${TARGET}-${VERSION} -t ${CONTEXT}/${IMAGE_NAME} -f ${DFILE} .
+	@if docker images ${CONTEXT}/${IMAGE_NAME}:${TARGET}-${VERSION}; then touch build; fi
+
+lint:
+	dockerfile_lint -f Dockerfile
+#	dockerfile_lint -f Dockerfile.rhel7
+
+openshift-test:
+	$(eval PROJ_RANDOM=test-$(shell shuf -i 100000-999999 -n 1))
+	oc login -u ${OC_USER} -p ${OC_PASS}
+	oc new-project ${PROJ_RANDOM}
+	oc new-project ${APB_APP}
+	oc adm policy add-role-to-user admin system:serviceaccount:${PROJ_RANDOM}:default
+	docker login -u ${OC_USER} -p ${OC_PASS} ${REGISTRY}:5000
+	docker tag ${CONTEXT}/${IMAGE_NAME}:${TARGET}-${VERSION} ${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME}
+	docker push ${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME}
+
+####### Test 1: Provision the "dev" plan (no storage)
+	oc project ${PROJ_RANDOM}
+	oc run ${IMAGE_NAME} --image=${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME} --restart=OnFailure -- provision -e namespace=${APB_APP} -e plan=dev
+# With --restart=OnFailure it creates a Job instead of a DC, so the following won't work:
+#	oc rollout status -w dc/${IMAGE_NAME}
+#	oc describe dc/${IMAGE_NAME}
+#	oc logs -f dc/${IMAGE_NAME}
+# FIXME: it's creating a pod, not a Job. Also fix the 'delete pod's below
+# FIXME: wait for the Job to complete with something better than a sleep/logs
```

Comment:
@codificat here's what i just did for hello-world-apb... moved to pod run and added logic to watch pod for completion...
https://github.com/fusor/apb-examples/commit/d84b7666cece64f481b17d6178203809697b7f21